platforms/karaf/commands/pom.xml

```diff
@@ -84,7 +84,7 @@
           <instructions>
             <Import-Package>
               !org.apache.felix.utils.properties,
-              org.apache.karaf.shell.api.action;version="[4,5)",
+              org.apache.karaf.shell.api.*;version="[4,5)",
               org.apache.karaf.shell.support.completers;version="[4,5)",
               org.slf4j.*;version="[1.6,2)",
```

Comment:
Can you conflate both lines into the following ?
```
             org.apache.karaf.shell.*;version="[4,5)",
```

Reply:
Sure, will be better !